examples/vault-prod-ready/README.md

```diff
@@ -0,0 +1,180 @@
+# Vault auto unseal cluster private with kms and acl for consul with encryption. PROD READY
+
+This example icombines the followings: Vault auto unseal - Vault private cluster - Consul with encryption and also adds some stuff that we expect in a prod ready working state, mainly:
+* Secure instances using security groups and remove [0.0.0.0/0]
+* Enable ACL on consul and pass the token in a secure way using AWS Secrets manager
+* Enable encryption on Consul (based on https://github.com/hashicorp/terraform-aws-consul/tree/master/examples/example-with-encryption)
+* Auto generate gossip_encryption_key and store it in AWS Secrets manager
+* Only allow ssh from Vpn server
```

Comment:
Not everyone who uses Vault will have a VPN server. Perhaps this should be a more generic list of security group IDs to whitelist? You could put the security group ID of a VPN server in that list, or a bastion host, or whatever else your organization is using.

Reply:
Agree